src/finiteVolume/processes/CMakeLists.txt

```diff
@@ -24,4 +24,5 @@ target_sources(ablateLibrary
         arbitrarySource.hpp
         buoyancy.hpp
         constantPressureFix.hpp
+        les.cpp
```

Comment:
Can you add les.hpp to the public section and move this les.cpp to the private section?